mycore-restapi/src/main/java/org/mycore/restapi/v1/utils/MCRRestAPIObjectsHelper.java

```diff
@@ -521,7 +525,7 @@ public static Response listObjects(UriInfo info, String format, String filter,
      * 
      * @see MCRRestAPIObjects#listDerivates(UriInfo, String, String, String)
      */
-    public static Response listDerivates(UriInfo info, String mcrObjID, String format,
+    public static Response listDerivates(UriInfo info, String mcrId, String format,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'listDerivates(UriInfo, String, String, String)' has an NPath complexity of 3240](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=5129212)